--- a/Lib/http/server.py
+++ b/Lib/http/server.py
@@ -1201,11 +1201,15 @@
             # Child
             try:
-                try:
-                    os.setuid(nobody)
-                except OSError:
-                    pass
+                if hasattr(os, "setuid"):
+                    try:
+                        os.setuid(nobody)
+                    except (PermissionError, OSError):
+                        pass
                 os.dup2(self.rfile.fileno(), 0)
                 os.dup2(self.wfile.fileno(), 1)
                 os.execve(scriptfile, args, env)
