diff --git a/Lib/ctypes/util.py b/Lib/ctypes/util.py
index 378f121..916805c 100644
--- a/Lib/ctypes/util.py
+++ b/Lib/ctypes/util.py
@@ -278,7 +278,7 @@ def _get_soname(f):
             # assuming GNU binutils / ELF
             if not f:
                 return None
-            objdump = shutil.which('objdump')
+            objdump = shutil.which('llvm-objdump')
             if not objdump:
                 # objdump is not available, give up
                 return None
@@ -315,7 +315,7 @@ def find_library(name):
             expr = os.fsencode(expr)
 
             try:
-                proc = subprocess.Popen(('/sbin/ldconfig', '-r'),
+                proc = subprocess.Popen(('@TERMUX_PREFIX@/bin/ldconfig', '-r'),
                                         stdout=subprocess.PIPE,
                                         stderr=subprocess.DEVNULL)
             except OSError:  # E.g. command not found
