--- a/Lib/ctypes/util.py
+++ b/Lib/ctypes/util.py
@@ -284,7 +284,7 @@
             # assuming GNU binutils / ELF
             if not f:
                 return None
-            objdump = shutil.which('objdump')
+            objdump = shutil.which('llvm-objdump')
             if not objdump:
                 # objdump is not available, give up
                 return None
@@ -321,7 +321,7 @@
             expr = os.fsencode(expr)

             try:
-                proc = subprocess.Popen(('/sbin/ldconfig', '-r'),
+                proc = subprocess.Popen(('@TERMUX_PREFIX@/bin/ldconfig', '-r'),
                                         stdout=subprocess.PIPE,
                                         stderr=subprocess.DEVNULL)
             except OSError:  # E.g. command not found
