diff --git a/Lib/platform.py b/Lib/platform.py
index 8895177e326..e02999739bf 100755
--- a/Lib/platform.py
+++ b/Lib/platform.py
@@ -619,7 +619,7 @@ else:
                    return buffer.value.decode("UTF-8", "backslashreplace")

            release = getprop("ro.build.version.release", release)
            api_level = @TERMUX_PKG_API_LEVEL@
            manufacturer = getprop("ro.product.manufacturer", manufacturer)
            model = getprop("ro.product.model", model)
            device = getprop("ro.product.device", device)
